src/plugins/intel_npu/src/plugin/npuw/compiled_model.hpp

```diff
@@ -25,7 +25,20 @@ namespace npuw {
 
 class InferRequest;
 
-class CompiledModel : public ov::ICompiledModel {
+class ICompiledModel : public ov::ICompiledModel {
+public:
+    ICompiledModel(const std::shared_ptr<ov::Model>& model, const std::shared_ptr<const ov::IPlugin>& plugin)
+        : ov::ICompiledModel(model, plugin) {}
+};
```

Comment:
what's the reason to have this? Is there anything common planned between `npuw::CompiledModel` and `npuw::LLMCompiledModel` ?

Reply:
Need to check this, probably not

Reply:
Fixed, left factory, removed new interface

Reply:
Fixed, left common interface, removed factory